packages/@ourworldindata/grapher/src/core/Grapher.tsx

```diff
@@ -1210,28 +1212,32 @@ export class Grapher
     @computed get currentTitle(): string {
         let text = this.displayTitle
         const selectedEntityNames = this.selection.selectedEntityNames
-        const showTitleAnnotation = !this.hideTitleAnnotation
+        const showEntityAnnotation = !this.hideAnnotationFieldsInTitle?.entity
+        const showTimeAnnotation = !this.hideAnnotationFieldsInTitle?.time
+        const showChangeInPrefix =
+            !this.hideAnnotationFieldsInTitle?.changeInPrefix
+
+        const isFacetingEnabled =
```

Comment:
```suggestion
        const isFacetingActive =
```

`isFacetingActive` is more expressive here.

Reply:
I thought `this.facetStrategy` refers to the facet strategy that is currently active? For example, if faceting is possible but currently only a single entity is selected, then `this.facetStrategy` would be `none` (currently not active) but `this.selectedFacetStrategy` would be set?

Reply:
Ah, right, I see now. Makes sense!